### PR TITLE
fix error overlay

### LINKED ISSUE #117: Error Overlay styling
The error indicator is positioned absolute which makes is very hard to see the indicator at all. Also because of the positioning it doesn't scroll.

problem

My suggestion would be to style it the following way which makes it very clear where the error is and also no special positioning is needed.

suggestion

I could formalize an PR if you approve the suggestion or have better ideas to visualize that case.

### PR DESCRIPTION
**Pull request by johann-sonntagbauer**
Error overlay now uses normal flow and inserts an additional line for error information. So no overlay of text and info caret occur any longer.

fixes #117

### CODE REVIEW

**Comment by CompuIves** [comment]
Wow that was quick! Thanks a lot!

Could you add yourself to the contributors? You can do this by running

```
npm run add
```

File: src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js [hunk 1]
```diff
@@ -8,15 +8,6 @@
...
-function removeNextBr(parent, component: ?Element) {
-  while (component != null && component.tagName.toLowerCase() !== 'br') {
-    component = component.nextElementSibling;
-  }
-  if (component != null) {
-    parent.removeChild(component);
-  }
-}
-
 function absolutifyCaret(component: Node) {
```
Comment (CompuIves) [inline src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js]: I think we could remove this whole function, right?

**Comment by CompuIves** [comment]
Just tested it, works great! The only standing thing is that I think we could delete `src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js` altogether.

File: src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js (hunk 1 above)
Comment (johann-sonntagbauer) [inline src/sandbox/react-error-overlay/utils/dom/absolutifyCaret.js]: you are right, no additional logic remains. Removed the file and cleaned up. Thx for the detailed review.

**Comment by CompuIves** [comment]
Great, thanks for the contribution!

**Pull request merged**